packages/containers/src/SubHeaderBar/SubHeaderBar.selectors.js

```diff
@@ -0,0 +1,22 @@
+import { DEFAULT_STATE, DISPLAY_NAME } from './SubHeaderBar.container';
+
+/**
+ * Selector on the state from the subheader bar id given.
+ * @param {object} state
+ * @param {string} idComponent
+ */
+export function getComponentState(state, idComponent) {
+	if (state.cmf.components.hasIn([DISPLAY_NAME, idComponent])) {
+		return state.cmf.components.getIn([DISPLAY_NAME, idComponent]);
```

Comment:
Can't you use `state.cmf.components.getIn([DISPLAY_NAME, idComponent], DEFAULT_STATE)` directly?